packages/core/resources/ec2_connect

```diff
@@ -28,7 +28,6 @@ _require_nolog() {
 
 _require() {
     _require_nolog "$@"
-    _log "$1=$2"
```

Comment:
instead of this, could change the relevant `_require` calls below to use `_require_nolog`